src/webui/www/private/views/rss.html

```diff
@@ -714,6 +720,21 @@
             });
         };
 
+        const editUrl = (path, url) => {
+            new MochaUI.Window({
```

Comment:
IIRC we can use modal dialogs now. This one maybe should use it.

@skomerko 
Any comments?

Reply:
There shouldn't be any issues but maybe @Piccirello found some. I can also convert all RSS dialog in one go once this is merged.

Reply:
The only thing that I believe may be unsolved is how to get keyboard shortcuts working for the modal (esc and enter). This isn't a huge blocker though so probably worth converting over.

Reply:
I looked into this a bit more. @skomerko since you'll be tackling the other RSS dialogs and have developed the pattern for this conversion, I'm going to leave this as is and let you do them all together. Thanks for assisting with this.

Reply:
Something I completely missed is that it looks like MUI.Modals are not meant to be stackable - you can open multiple but overlay is not readjusted at all and each modal has the exact same z-index. Fortunately, it's not the case with MUI.Window which properly assigns z-index values and fires onfocus callback in all circumstances. So I think I'll simply add overlay handling to base window options to achieve the same effect and switch MUI.Modals back to MUI.Windows (fortunately there's not much to change really, just a couple of lines).